core/audits/network-requests.js

```diff
@@ -111,8 +118,8 @@ class NetworkRequests extends Audit {
     const tableDetails = Audit.makeTableDetails(headings, results);
 
     // Include starting timestamp to allow syncing requests with navStart/metric timestamps.
-    const networkStartTimeTs = Number.isFinite(earliestStartTime) ?
-        earliestStartTime * 1000 : undefined;
+    const networkStartTimeTs = Number.isFinite(earliestMainThreadStartTime) ?
+        earliestMainThreadStartTime * 1000 : undefined;
```

Comment:
> i'm not sure about this `* 1000`, given that rendererstartime should be in ms.

Yeah, I guess it's debatable what it should use. My original thinking was it should use microseconds because you compare it against `observedNavigationStartTs` to get the offset between CPU task timestamps and network times:

```sql
(CAST(JSON_VALUE(lighthouse, '$.audits."network-requests".details.debugData.networkStartTimeTs') AS FLOAT64) - CAST(JSON_VALUE(lighthouse, '$.audits.metrics.details.items[0].observedNavigationStartTs') AS FLOAT64)) / 1000 AS network_ts_offset
```

I put the `Ts` at the end of `networkStartTimeTs` to try to communicate that, but it's also a good argument that it's simpler if this audit only had values in ms 🤷